Intersect.Client.Framework/Gwen/Control/ScrollControl.cs

```diff
@@ -258,7 +258,8 @@ protected override bool OnMouseHWheeled(int delta)
             if (CanScrollH && mHorizontalScrollBar.IsVisible)
             {
                 if (mHorizontalScrollBar.SetScrollAmount(
-                    mHorizontalScrollBar.ScrollAmount - mHorizontalScrollBar.NudgeAmount * (delta / 60.0f), true
+                    mHorizontalScrollBar.ScrollAmount - mHorizontalScrollBar.NudgeAmount * (delta / 60.0f), 
```

Comment:
I meant for this calculation to be split out into a temporary variable, and the temporary variable used in the function call.

Reply:
Oh, fair enough :P